msteams-platform/task-modules-and-cards/cards/cards-actions.md

```diff
@@ -698,11 +698,11 @@ The following code shows an example of Adaptive Cards with `invoke` action with
 ```json
 {
   "type": "Action.Submit",
-  "title": "submit"
+  "title": "submit",
   "data": {
     "msteams": {
       "type": "invoke",
-      "Value": "some value"
+      "value": "{ \"string\": \"object\"}"
```

Comment:
This can be either stringified JSON or just JSON plain object. Those 2 values are all valid:
1. `"value": "{ \"string\": \"object\"}"`
2. ```"value": { "key": 1234 }```